swift/WorkflowUI/Sources/Container/ContainerViewController+AnyWorkflow.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import Workflow
+
+extension ContainerViewController {
```

Comment:
You’ll need to wrap it all in `#if canImport(UIKit)` since ContainerViewController has that.

Reply:
Done!